### ISSUE TITLE
ulnar positive variance

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10750]
**Preferred term label:**
ulnar positive variance

**Synonyms**

**Definition (free text, please give PubMed ID)**
AP wrist X-ray showing the distal ulna articular surface more distal than the distal radius articular surface

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Ulnocarpal abutment syndrome

**Your nano-attribution (ORCID)**

**Comment by MickeySegal** [comment]
Also need "ulnar negative variance" with synonym "AP wrist X-ray showing the distal ulna articular surface more proximal than the distal radius articular surface".  Seen in Kienböck disease.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001421
- label: Ulnar positive variance
- parents: [Abnormal forearm morphology (HP:0002973)](https://hpo.jax.org/app/browse/term/HP:0002973)
- pmid: PMID:36752828
- def: A phenomenon whereby the distal ulna articular surface is more distal than the distal radius articular surface.
- comment: The triangular fibrocartilage complex (TFCC) is a well-defined anatomical entity of the wrist joint known as the primary stabilizing structure of the distal radioulnar joint (DRUJ). The TFCC provides considerable DRUJ stability. The instability of DRUJ caused by the foveal tear of the TFCC also results in an abnormal anatomic relationship between the radius and ulna. When the foveal attachment of the TFCC is disrupted, the ulnar head is free to sublux dorsally relative to the sigmoid notch of the radius, which is aggravated under the stress of pronation. This change in the anatomic relationship between the radius and ulna can be easily observed on plain radiographs.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10750
- synonyms:

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001422
- label: Negative ulnar variance
- parents: [Abnormal forearm morphology (HP:0002973)](https://hpo.jax.org/app/browse/term/HP:0002973)
- pmid: 
- def: Negative ulnar variance is a condition in which the ulna is relatively shorter than the radius at the carpus. AP wrist X-ray shows the distal ulna articular surface more proximal than the distal radius articular surface.
- comment: Negative ulnar variance may be observed in Kienbock disease.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10750
- synonyms:
